iothub/service/src/Jobs/ScheduledJob.cs

```diff
@@ -27,35 +27,68 @@ public class ScheduledJob
         /// <summary>
         /// Scheduled job start time in UTC.
         /// </summary>
-        [JsonProperty(PropertyName = "createdTime", NullValueHandling = NullValueHandling.Ignore)]
+        [JsonProperty(PropertyName = "createdDateTimeUtc", NullValueHandling = NullValueHandling.Ignore)]
         public DateTime? CreatedTimeUtc { get; internal set; }
 
+        // Some service Jobs APIs use "createdTime" as the key for this value and some others use "createdDateTimeUtc".
+        // This private field is a workaround that allows us to deserialize either "createdTime" or "createdDateTimeUtc"
+        // as the created time value for this class and expose it either way as CreatedTimeUtc.
+        [JsonProperty(PropertyName = "createdTime")]
```

Comment:
Does this need any null value handling configuration?

Reply:
I'll add the config to make it ignore if null